server.py

```diff
@@ -278,7 +278,12 @@ def pbs_history(passband_request='all'):
             continue
 
         pb = phoebe.atmospheres.passbands.Passband.load(fname, load_content=False)
-        pb_history[pbr] = pb.history
+        # exporting a list to dict:
+        history = dict()
+        for entry in pb.history:
+            timestamp, message = entry.split(': ')
+            history[timestamp] = message
+        pb_history[pbr] = history
```

Comment:
do we only want to do this for phoebe before 2.5?

Reply:
nope, because all served passbands follow the new structure.

Reply:
I know, but should newer versions of phoebe just be updated to handle the native structure directly and not be translated by the server?

Reply:
maybe its just more convenient to keep the dictionary format though and then there aren't any changes as far as the user is concerned

Reply:
> I know, but should newer versions of phoebe just be updated to handle the native structure directly and not be translated by the server?

That's what the [other PR](https://github.com/phoebe-project/phoebe2/pull/979) is for.